.github/workflows/ci.yml

```diff
@@ -0,0 +1,118 @@
+name: ci
+
+on: [push, pull_request]
+
+env:
+  X_PYTHON_VERSION: "3.10"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.10"]
+
+
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install coverage
+        cd ./sdk
+        pip install .
```

Comment:
Have you checked that this command works and installs all necessary requirements?

Reply:
I forgot to make a merge request of the branch "pyproject_toml". This branch was based on pyproject_toml. Should I make a new PR to merge "pyproject_toml" first?

Reply:
Yes, if this depends on that, then it certainly makes sense to merge it first.